docs/REST API/rest-api.md

```diff
@@ -2930,7 +2932,8 @@ Returns the information of the specified host.
         "resolution": "fix with these other steps...",
         "platform": "darwin",
         "response": "fail",
-        "critical": false
+        "critical": false,
+        "fleet_maintained": false
```

Comment:
Dev note on error messages when attempting to perform actions on Fleet-maintained policies:

```
Can't delete policies. Policy with ID 3 is a Fleet-maintained policy. Fleet-maintained policies can't be deleted.
```

If the user tries to edit the policy via the [Edit policy (global)](https://fleetdm.com/docs/rest-api/rest-api#edit-policy) or [Edit team policy](https://fleetdm.com/docs/rest-api/rest-api#edit-team-policy) API endpoints, show this error:

```
Can't edit Fleet-maintained policies.
```